bb/src/main/java/com/richardmcdougall/bb/RFClientServer.java

```diff
@@ -719,6 +721,8 @@ public void sendRemote(int cmd, long value, int type) {
         int16ToPacket(clientPacket, cmd);
         // Value
         int32ToPacket(clientPacket, value);
```

Comment:
Would you mind if we put the address first -- just because that's what we've done for all other packet formats (magic header + address)

Reply:
ack